store/store_test.go

```diff
@@ -64,6 +64,10 @@ type configTestSuite struct{}
 
 var _ = Suite(&configTestSuite{})
 
+// this is what snap.E("1*") looks like when decoded into an interface{}
+var iOneStar = map[string]interface{}{"read": []interface{}{0., 1.}, "write": []interface{}{1.}}
```

Comment:
What do `I` and `e` stand for in the two variables?

Reply:
i -> interface, e -> epoch :man_shrugging: